tslint.json

```diff
@@ -109,3 +109,4 @@
     "directive-class-suffix": true
   }
 }
+
```

Comment:
Why is this unnecessary blank line is added?
It doesn't look like a missing newline on EOL fix like https://github.com/fossasia/loklak_search/pull/208/files#diff-c0745258b9513d87ab38c11820efcdfbR95